docs/my-website/docs/observability/opentelemetry_integration.md

```diff
@@ -31,6 +31,16 @@ OTEL_HEADERS="Authorization=Bearer%20<your-api-key>"
 
 </TabItem>
 
+<TabItem value="laminar" label="Log to Laminar">
+
+```shell
+OTEL_EXPORTER="otlp_grpc"
+OTEL_ENDPOINT="https://api.lmnr.ai:8443"
+OTEL_HEADERS="authorization=Bearer <project-api-key>"
+```
+
+</TabItem>
+
```

Comment:
please put it after HTTP and GRPC collector

Reply:
Done

![Screenshot 2024-11-08 at 21 00 50](https://github.com/user-attachments/assets/8c583483-3703-4dcb-8dd1-ee609f30f9ff)